src/Microsoft.DotNet.Build.Tasks.net45/Microsoft.DotNet.Build.Tasks.net45.csproj

```diff
@@ -39,4 +39,25 @@
     <TargetingPackReference Include="System.Xml" />
     <TargetingPackReference Include="System.Xml.Linq" />
   </ItemGroup>
+  <Target Name="AfterBuild">
+    <!-- Manually binplace DiaSymReader runtimes for win7-x86, which the official builds use. -->
```

Comment:
Is this worth preemptively logging an issue for?  Then the comment can refer back to it and we have an idea of the work that has to happen to remove the hack.